packages/peregrine/package.json

```diff
@@ -33,8 +32,8 @@
     "@babel/plugin-syntax-dynamic-import": "~7.2.0",
     "@babel/plugin-syntax-jsx": "~7.2.0",
     "@babel/plugin-transform-react-jsx": "~7.3.0",
-    "@babel/plugin-transform-runtime": "~7.3.4",
-    "@babel/runtime": "~7.3.4",
+    "@babel/plugin-transform-runtime": "~7.4.4",
+    "@babel/runtime": "~7.4.2",
```

Comment:
Since it's listed in `peerDependencies` do we need `@babel/runtime` here?

Reply:
Yeah, for development.

Reply:
@supernova-at  Entries in `peerDependencies` aren't installed when you run `yarn` (or `npm install`). They're needed for tests, though, so we have to list them in `devDependencies` as well.